tools/apigee-x-trial-provision/apigee-x-trial-provision.sh

```diff
@@ -315,11 +315,29 @@ gcloud compute forwarding-rules create apigee-proxy-https-lb-rule \
 
 set -e
 
-# TODO: [ ] wait till LB is ready
 echo ""
-echo "Almost done. It take some time (another 5-8 minutes) to provision load balancer infrastructure."
+echo "Almost done. It take some time (another 5-8 minutes) to provision the load balancer infrastructure."
 echo ""
 
+# TODO: more intelligent wait until LB is ready
+
+set +e
```

Comment:
i think we would rather do 

```
cmd || echo "command failed, but we are continuing"
``` 
or something on individual lines rather than +e a whole block.

Bonus points for not just doing `|| true` but checking the specific reason it failed matches what you expect and will allow

Reply:
fixed